README.md

```diff
@@ -96,6 +96,8 @@ A fragment is an http(s) server that renders only the part of the page and sets
 
 A JavaScript of the fragment is an AMD module, that exports an `init` function, that will be called with DOM element of the fragment as an argument.
 
+A fragment also can not return a redirect `Location:` header, as Tailor is not made to follow redirects. That is on purpose as redirects can introduce unwanted latency.
```

Comment:
I would rename it as `Tailor will not follow redirects even if fragment response contains 'Location' Header,` and also mention the primary fragment exception as I commented.

Reply:
I have updated it 😄 Good suggestions!